File: j-LIB-core/src/main/java/io/github/dot166/jlib/app/jLIBCoreApp.java
package io.github.dot166.jlib.app;

import android.app.Application;

import com.google.android.material.color.DynamicColors;

public class jLIBCoreApp extends Application {
    public static final String TAG = "jLIB";

    public void onCreate() {
        super.onCreate();
        DynamicColors.applyToActivitiesIfAvailable(this);
    }
}

File: j-LIB-core/src/main/java/io/github/dot166/jlib/app/jLIBCoreApp.kt
package io.github.dot166.jlib.app

import android.app.Application
import com.google.android.material.color.DynamicColors

open class jLIBCoreApp : Application() {
    override fun onCreate() {
        super.onCreate()
        DynamicColors.applyToActivitiesIfAvailable(this)
    }
}
